stake-pool/cli/src/main.rs

```diff
@@ -2426,6 +2437,12 @@ fn main() {
                     .takes_value(false)
                     .help("Do not automatically merge transient stakes. Useful if the stake pool is in an expected state, but the balances still need to be updated."),
             )
+            .arg(
```

Comment:
Can you just update the `.help` for `force` to say "Update balances" rather than "Update all balances", since now we have some functionality that may not update _all_ of them?